pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -125,9 +124,31 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 
 	// setup security context
 	effectiveSc := securitycontext.DetermineEffectiveSecurityContext(pod, container)
+
+	// Strip down all the unnecessary options on the Windows
+	if effectiveSc.SELinuxOptions != nil {
+		effectiveSc.SELinuxOptions = nil
+	}
+
+	if effectiveSc.AllowPrivilegeEscalation != nil {
+		effectiveSc.AllowPrivilegeEscalation = nil
+	}
+
+	if effectiveSc.Capabilities != nil {
+		effectiveSc.Capabilities = nil
+	}
+
+	if effectiveSc.Privileged != nil {
+		effectiveSc.Privileged = nil
+	}
+
+	if effectiveSc.ProcMount != nil {
+		effectiveSc.ProcMount = nil
+	}
```

Comment:
given the e2e test doesn't test every one of these scenarios, I would suggest add unit tests here as well